firefox-ios/Storage/DefaultSuggestedSites.swift

```diff
@@ -80,7 +80,7 @@ open class DefaultSuggestedSites {
                     forRemoteResource: URL(string: "https://abs.twimg.com/responsive-web/client-web/icon-ios.77d25eba.png")!
                 ),
                 trackingId: 628,
-                title: .DefaultSuggestedX
+                title: "x.com"
```

Comment:
Oops! Make sure we don't change the Twitter title here from our translated string.

Reply:
Yes sorry @ih-codes forgot to restore it, not it is good